dist/stat_data.py

```diff
@@ -518,11 +518,20 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Dhandle statistics
     ##########################################
+    DhandleStat('dh_conn_handle_btree_count', 'B-tree connection data handles currently active', 'no_clear,no_scale'),
```

Comment:
This spelling of btree is inconsistent with other statistics which simply say "btree", please make it consistent.